sources/web/datalab/static/notebook-app.js

```diff
@@ -95,6 +95,9 @@ define(['static/appbar', 'static/minitoolbar'], function(appbar, minitoolbar) {
       })
     });
 
+    idleTimeout.setupKernelBusyHeartbeat();
+    $('#mainArea').scroll(idleTimeout.notebookScrolled);
```

Comment:
I think we'll need a different div to trigger this event in the non-notebook pages